src/Tests/dotnet-watch.Tests/Watch/Utilities/WatchableApp.cs

```diff
@@ -121,7 +121,7 @@ public void Start(string projectRootPath, IEnumerable<string> arguments, string
             TestFlags testFlags = TestFlags.RunningAsTest,
             [CallerMemberName] string name = null)
         {
-            var args = new[] { "run", "--" };
+            var args = new[] { "run", "--verbose", "--" };
```

Comment:
Did you want this change committed? I'm not familiar yet on what the output would do to the test run logs.